lib/clamp-embed.in

```diff
@@ -16,21 +16,12 @@ OPT=$BINDIR/opt
 LLC=$BINDIR/llc
 LIB=$BINDIR/../lib
```

Comment:
I guess you won't need BINDIR, OPT, LLC, and LIB since removing the cpu-rename if block

Reply:
True.